conf/documentation.conf

```diff
@@ -534,6 +534,24 @@ You can specify the same domain with a different port multiple times and they wi
 The configuration parameter "passthrough" must be enabled for passthroughs to be effective.
 EOT
 
+[fencing.isolation_passthrough]
```

Comment:
Can you adjust the passthough configuration to make it explicit that they are only honored in registration